src/test/java/fi/aalto/cs/apluscourses/utils/StateMonitorTest.java

```diff
@@ -61,7 +59,7 @@ public void testSetConditionallyReturnsFalse() {
     int expectedState = 20;
     int newState = 220;
     assertFalse("setConditionally() should return false",
-        stateMonitor.setConditionally(expectedState, newState));
+        stateMonitor.setConditionallyTo(newState, expectedState));
```

Comment:
should the method name here and across the class also be renamed into `testSetConditionally`**To**`ReturnsFalse()`?

Reply:
You're right.